mailbox/plugin/spamassassin/src/main/java/org/apache/james/mailbox/spamassassin/SpamAssassinListener.java

```diff
@@ -94,6 +93,16 @@ public void event(Event event) {
         }
     }
 
+    private boolean isAppendedToInbox(EventFactory.AddedImpl addedEvent) {
+        try {
+            return systemMailboxesProvider.findMailbox(Role.INBOX, addedEvent.getSession())
```

Comment:
`==` is probably not ok, it shoud be `.equals` ?

Reply:
Good catch